infra/concrexit/Dockerfile

```diff
@@ -52,9 +74,9 @@ ENV PATH=/venv/bin:$PATH \
     SENDFILE_ROOT="/volumes/media"
 
 # Install runtime dependencies.
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-    postgresql-client libjpeg-dev zlib1g-dev libwebp-dev libmagic1 && \
+RUN apk update && \
+    apk add \
+    postgresql-client jpeg-dev zlib-dev libwebp-dev libmagic runuser && \
     rm -rf /var/lib/apt/lists/*
```

Comment:
the apk cache isn't at `/var/lib/apt/lists` any more. For apk you can use --no-cache, or maybe buildkit's `RUN --mount=type=cache` (https://docs.docker.com/engine/reference/builder/#run---mounttypecache) to get better performance.